src/validations/rules/ssp.sch

```diff
@@ -2396,9 +2396,9 @@
                         ../oscal:control-implementation/oscal:implemented-requirement[@control-id eq 'si-8']//*[matches(., 'SPF')] and
                         ../oscal:control-implementation/oscal:implemented-requirement[@control-id eq 'si-8']//*[matches(., 'DKIM')])
                         then
-                            (true())
+                            (false())
                         else
-                            (false()))
+                            (true()))
                     else
                         (true())">This FedRAMP SSP references 'email', 'e-mail', or 'electronic mail'.</sch:assert>
```

Comment:
This positive test result assertion message (**_This FedRAMP SSP references 'email', 'e-mail', or 'electronic mail'._**)remains incorrect. It does not address the absence within the SSP of any mention of email.

Please see previous comments regarding positive test outcome assertion messages.